rllib/env/single_agent_env_runner.py

```diff
@@ -685,13 +689,16 @@ def make_env(self) -> None:
             )
         gym.register("rllib-single-agent-env-v0", entry_point=entry_point)
 
-        # Wrap into `VectorListInfo`` wrapper to get infos as lists.
-        self.env: gym.Wrapper = gym.wrappers.VectorListInfo(
-            gym.vector.make(
-                "rllib-single-agent-env-v0",
-                num_envs=self.config.num_envs_per_env_runner,
-                asynchronous=self.config.remote_worker_envs,
-            )
+        # Note that `infos` will be returned as vectors as well. Since the `gym.wrapper.
+        # VectorListInfo` wrapper currently does not work (throws an error on
+        # construction), we'll do this relatively simple conversion here in the
+        # EnvRunner on our own.
+        async_ = self.config.remote_worker_envs
+        self.env: gym.Wrapper = gym.make_vec(
```

Comment:
Btw. we get constantly a warning from the `gymnasium` registry that an env with this name already exists and will be overridden. We should find a way to avoid this warning. There are already enough warnings printed to the user.

Reply:
Great point! I'll see what I can do here. Maybe check, before the env is being registered, if it hasn't been registered already. ...

Reply:
@sven1977 my guess is that this happens when multiple runners are used. Maybe we just use the runner ID or something unique for each env registration.